reactive/kubernetes_worker.py

```diff
@@ -452,14 +463,28 @@ def watch_for_changes():
         set_state('kubernetes-worker.restart-needed')
 
 
+@when('kubernetes-worker.cloud.pending',
+          'kubernetes-worker.cloud.blocked')
```

Comment:
The cloud provider can't be both blocked and pending.  Also, why does the container runtime depend on having a cloud integrator attached?

Reply:
Ah, that's my mistake.  Fixed.